project-koku/koku
# [COST-4117] - Skipping NaN values

COST-4117 Jira Ticket

[COST-4117](https://issues.redhat.com/browse/COST-4117)

## Description

This change will skip over NaN resource_ids and Tags during ocp on Azure/AWS matching.

## Testing

1. Checkout Branch
2. Restart Koku
3. Load OCP data
4. Load Azure data with at least 1 matching resource id to an OCP node + a bunch of NaN resource id values
5. Ingest data successfully.
6. Do the same thing on main and see log errors

`Unknown processor error: Can only use .str accessor with string values!'`

## Notes
I actually tested this with some partial customer data. Not sure how easy this would be to craft with Nise.
...
